src/pages/CreateCompanyPage.jsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from "react";
+import { Navigate } from "react-router-dom";
+import { Header } from "../components/Header/Header";
+import { Form } from "../components/Form/Form";
+import { SubmitButton } from "../components/Form/SubmitButton";
+import { Button } from "../components/Form/Button";
+import { useNavigate } from "react-router-dom";
+
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+import { CompanyTextInput } from "../components/Form/CompanyTextInput";
+import { createCompany } from "../apiCalls/createCompany";
+
+const schema = yup.object().shape({
```

Comment:
Feel free to extract schemas to separate file
```suggestion
const SCHEMA = yup.object().shape({
```
```suggestion
const schema = yup.object().shape({
```